collection-scripts/gather_feastoperator

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+source common.sh
+resources=("feastoperator")
```

Comment:
is this correct?

Reply:
I thought it was at least 😅

The component creates a single clusterwide resource `feastoperator` with the name `default-feastoperator`. 

`get_operator_resource()` from common.sh  should result in an `oc adm inspect feastoperator/default-feastoperator --dest-dir "$DST_DIR"` to the must-gather.

Reply:
here should be the CR from feast component "featurestores"